### DISCUSSION TITLE
How to dpkg -Purge everything

### DISCUSSION THREAD

**Discussion by rlovell3** [AlexandreRouma/SDRPlusPlus#739] [Q&A] [answered]
I'm going backwards, and need some help:
Ubuntu(PopOS 21.10) using nooelec smart (also tried rtl-sdr blog unit).  After installing numerous dependencies, I finally had basic FM radio stations playing.  After bumping a few settings in sdrpp, and ultra destroying items in the Module Manager, I've lost even the ability to hear those same FM stations.  So I wanted to uninstall and start over.  After running a {sudo dpkg -P sdrpp} operation to uninstall sdrpp, the sdrpp application was confirmed gone.  A quick reinstall, and I note that all the various tweaks (including saved FM radio stations) and the terrible mangling I had done before showed up.  I obviously did not purge all the config files.  What do I need to do to fully purge and start from zero?  Thanks.

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
The config is in `~/.config/sdrpp`. Deleting this folder should be the only thing needed to reset sdrpp.

It looks like dpkg only deletes config files if the software package instructs it to. See `-P` in the dpkg man page for details.
